src/rules/no-native-attributes.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * @fileoverview Disallows use of native attributes as properties
+ * @author Pascal Schilp <https://github.com/thepassle>
+ */
+
+import { Rule } from 'eslint';
+import * as ESTree from 'estree';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+const rule: Rule.RuleModule = {
+  meta: {
+    docs: {
+      description: 'Disallows use of native attributes as properties',
+      recommended: false,
+      url: 'https://github.com/43081j/eslint-plugin-lit/blob/master/docs/rules/no-native-attributes.md'
+    },
+    schema: [],
+    messages: {
+      noNativeAttributes:
+        'Avoid using global native attributes as properties, ' +
```

Comment:
would be nice to have context, something like:

> The {{ attr }} attribute is a native global attribute. Using it as a property could have unintended side-affects

then you can specify the `attr` when reporting an issue by doing something like:

```ts
report({
  messageId: 'noNativeAttributes',
  data: {attr}
});
```